stock_picking_group_by_partner_by_carrier/migrations/16.0.2.0.0/post-migrate.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2024 ACSONE SA/NV (https://www.acsone.eu)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+import logging
+
+from openupgradelib import openupgrade
+
+_logger = logging.getLogger(__name__)
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    # Declare procurement group linked to more than one sale order as
+    # merged for procurement groups linked to stock pickings of type outgoing
+    SQL = """
+        UPDATE
+            procurement_group
+        SET
+            is_merged = TRUE
+        WHERE
+            id IN (
+                SELECT
+                    procurement_group_id
+                FROM
+                    procurement_group_sale_order_rel
+                    JOIN stock_picking ON stock_picking.group_id = procurement_group_id
+                WHERE
+                    stock_picking.state NOT IN ('cancel', 'done')
+                    AND stock_picking.picking_type_id = 4
```

Comment:
Is it reliable for all instances to use `4` as `picking_type_id` here and below ?
Makes more sense to  search for the xmlid to me

Reply:
@TDu The code is fixed now. Thank you for the review.